omegaconf/grammar_visitor.py

```diff
@@ -350,16 +350,43 @@ def _unescape(
         resolving of the interpolation).
         """
         chrs = []
-        for node in seq:
+        for node, next_node in zip_longest(seq, seq[1:]):
```

Comment:
What does this do on the last node?
do you get another iteration with:
```
node: last node in the sequence
next_node : none?
```

Or is it skipping the last node?
If it's skipping, is there anything we need to unescape there?

Reply:
The first option (this is why I'm using [`zip_longest`](https://docs.python.org/3/library/itertools.html#itertools.zip_longest) instead of `zip` -- this was the most concise way I found)

Reply:
Nice , never used zip_longest.
My immediate intuition was to actually iterate on 1 to n, and consider unescaping the previous token.
But it sounds equivalent.